integration-tests/__tests__/__snapshots__/globals.test.js.snap

```diff
@@ -32,7 +32,6 @@ exports[`cannot test with no implementation 1`] = `
       4 |     test('test, no implementation');
       5 |   
 
-      at packages/jest-jasmine2/build/jasmine/Env.js:<<LINE>>:<<COLUMN>>
       at __tests__/only-constructs.test.js:<<LINE>>:<<COLUMN>>
```

Comment:
pretty sure we don't need the `LINE` `COLUMN` stuff now that we have proper source maps. not relevant to this PR, though.

Reply:
removed anyway, thanks